src/test/java/org/example/homework13/blackjack/DeckTest.java

```diff
@@ -0,0 +1,44 @@
+package org.example.homework13.blackjack;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DeckTest {
+
+    @Test
+    void drawCard() {
+        Deck deck = new Deck();
+        Card card = deck.drawCard();
+        assertEquals(Rank.TWO, card.getRank());
+        assertEquals(Suit.CLUBS, card.getSuit());
+    }
+
+    @Test
+    void shuffle() {
+        Deck referenceDeck = new Deck();
+        Deck shuffledDeck = new Deck();
```

Comment:
no sense to create 2 Decks. shuffle method changes the state of the field `cards` of the Deck instance. so it is better to check this field before shuffle and after it
So you should check cards list before shuffle and cards list after shuffle

Reply:
I had to add a getter for cards list. Updated to use a copy of list as reference